src/components/manage/Contents/ContentsItem.jsx

```diff
@@ -4,15 +4,48 @@
  */
 
 import React from 'react';
-import { Icon, Table, Dropdown } from 'semantic-ui-react';
+import { Table, Dropdown } from 'semantic-ui-react';
 import { Link } from 'react-router-dom';
 import PropTypes from 'prop-types';
 import { map } from 'lodash';
 import moment from 'moment';
 import { DragSource, DropTarget } from 'react-dnd';
 import { FormattedMessage } from 'react-intl';
+import { Icon } from '@plone/volto/components';
+import moreSVG from '@plone/volto/icons/more.svg';
+import addDocumentSVG from '@plone/volto/icons/add-document.svg';
+import linkSVG from '@plone/volto/icons/link.svg';
+import calendarSVG from '@plone/volto/icons/calendar.svg';
+import folderSVG from '@plone/volto/icons/folder.svg';
+import fileSVG from '@plone/volto/icons/file.svg';
+import imageSVG from '@plone/volto/icons/image.svg';
+import checkboxUncheckedSVG from '@plone/volto/icons/checkbox-unchecked.svg';
+import checkboxCheckedSVG from '@plone/volto/icons/checkbox-checked.svg';
+import cutSVG from '@plone/volto/icons/cut.svg';
+import deleteSVG from '@plone/volto/icons/delete.svg';
+import copySVG from '@plone/volto/icons/copy.svg';
+import showSVG from '@plone/volto/icons/show.svg';
+import moveUpSVG from '@plone/volto/icons/move-up.svg';
+import moveDownSVG from '@plone/volto/icons/move-down.svg';
+import editingSVG from '@plone/volto/icons/editing.svg';
+import dragSVG from '@plone/volto/icons/drag.svg';
 
-import { getIcon } from '@plone/volto/helpers';
+export function getIcon(type, isFolderish) {
```

Comment:
Why we had to modify this and not use the helpers one?

Reply:
Because we are not using semantic-ui icon class. I think that I should modify the helper function also but I didn't do that because it might break the app which uses the same function for getting icons.